### PR TITLE
Cleaned up Openstack config options.

### PR DESCRIPTION
**Pull request by varikin**
Renamed tenant to project, the preferred term in the current Openstack
verison. Removed the master option as no one can remember what it was for.

Removed master property from openstack config.

### CODE REVIEW

**Comment by duftler** [comment]
LGTM. Needs a rebase.

**Comment by varikin** [comment]
Hold off on this a while. I am going to make one more change. The `endpoint` option is poorly named.

**Comment by varikin** [comment]
OK, got the endpoint option changed.

File: clouddriver-web/config/clouddriver.yml [hunk 1]
```diff
@@ -256,18 +256,19 @@ google:
...
-      tenantName: ${OS_TENANT_NAME:''}
+      projectName: ${OS_PROJECT_NAME:''}
       domainName: Default
-      endpoint: ${OS_AUTH_URL:''}
+      authUrl: ${OS_AUTH_URL:''}
       regions: ${OS_REGIONS:''}
       insecure: ${OS_INSECURE:false}
-      heatTemplatePath: /path/to/custom/heat/template(s)
+      # Replacing the Heat template is not recommended
+      # The implemnation is tightly coupled with the suppplied template
```
Comment (duftler) [inline clouddriver-web/config/clouddriver.yml]: s/implemnation/implementation

**Comment by lwander** [comment]
Ready to merge?

**Comment by varikin** [comment]
@duftler already merged it :)

**Comment by varikin** [comment]
And now I see my email notifications are coming way out of order

**Pull request merged**